src/woo_publications/metadata/tests/test_informatie_categorie_admin.py

```diff
@@ -0,0 +1,212 @@
+                str(response.content),
+            )
+
+        with self.assertRaisesMessage(
+            AssertionError,
+            "No field by the name 'identifier' found (fields: 'csrfmiddlewaretoken')",
+        ):
+            form["identifier"] = "https://www.example.com/waardenlijsten/12"
+
+    def test_informatie_category_admin_can_update_item_with_oorsprong_zelf_toegevoegd(
+        self,
+    ):
+        url = reverse(
+            "admin:metadata_informatiecategorie_change",
+            kwargs={"object_id": self.information_category_custom_entry.id},
+        )
+
+        response = self.app.get(url, user=self.user)
+        form = response.forms["informatiecategorie_form"]
+
+        self.assertEqual(response.status_code, 200)
+        # page has save button.
+        self.assertInHTML(
+            '<input type="submit" value="Opslaan" class="default" name="_save">',
+            str(response.content),
+        )
+
+        # test if identifier isn't editable
+        with self.assertRaises(AssertionError):
+            form["identifier"] = "https://www.example.com/waardenlijsten/12"
+
+        form["naam"] = "changed"
+        form["naam_meervoud"] = "changed"
+        form["definitie"] = "changed"
+        response = form.submit("submit")
+
+        self.assertEqual(response.status_code, 302)
+        self.information_category_custom_entry.refresh_from_db()
+
+        self.assertEqual(self.information_category_custom_entry.naam, "changed")
+        self.assertEqual(
+            self.information_category_custom_entry.naam_meervoud, "changed"
+        )
+        self.assertEqual(self.information_category_custom_entry.definitie, "changed")
+
+    def test_informatie_category_admin_create_item(self):
+        response = self.app.get(
+            reverse("admin:metadata_informatiecategorie_add"), user=self.user
+        )
+        form = response.forms["informatiecategorie_form"]
+
+        form["identifier"] = "https://www.example.com/waardenlijsten/999"
+        form["naam"] = "new item"
+        form["naam_meervoud"] = "new items"
+        form["definitie"] = (
+            "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris risus nibh, "
+            "iaculis eu cursus sit amet, accumsan ac urna. Mauris interdum eleifend eros sed consectetur."
+        )
+
+        form.submit("submit")
+        self.assertTrue(
+            InformatieCategorie.objects.filter(
+                identifier="https://www.example.com/waardenlijsten/999",
+                naam="new item",
+                naam_meervoud="new items",
+                definitie=(
+                    "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris risus nibh, "
+                    "iaculis eu cursus sit amet, accumsan ac urna. Mauris interdum eleifend eros sed consectetur."
+                ),
+                oorsprong=InformatieCategorieOrigins.custom_entry,
+                order=2,
+            ).exists()
+        )
```

Comment:
when this breaks, it's going to suck to figure out which aspect specifically is broken, so better is:

```py
added_item = InformationCategory.objects.order_by("-pk").first()
assert added_item is not None
self.assertEqual(added_item.naam, "new item")
self.assertTrue(added_item.identifier.startswith(CUSTOM_IDENTIFIER_URL_PREFIX))
self.assertEqual(added_item.naam_meervoud, ...)
...  # etc.
```
so when something fails, you know exactly which bit and can investigate that much more efficiently